build.gradle.kts

```diff
@@ -1,6 +1,5 @@
 plugins {
     java
-    application
```

Comment:
@KotlinIsland was this plugin being used for anything?

Reply:
nah